armsrc/mifareutil.h

```diff
@@ -57,6 +57,10 @@
 
 #define cardSTATE_TO_IDLE() cardSTATE = MFEMUL_IDLE; LED_B_OFF(); LED_C_OFF();
 
+#ifndef MifareBlockToSector
+#define MifareBlockToSector(block) (block < 128 ? block / 4 : (block - 128) / 16 + 32)
```

Comment:
https://github.com/RfidResearchGroup/proxmark3/blob/master/client/mifare/mifare4.h#L64

Reply:
'mifare4.h' is a part of client, Not good idea to include this part into 'armsrc'

Reply:
ok